docs/search.md

```diff
@@ -37,6 +37,63 @@ curl -XGET 'https://…/api/v1/search' -d '{
 }'
 ```
 
+## Advanced Search 
+
+Advanced Search is based on [Elasticsearch](https://www.elastic.co/). It comprises of spatio-temporal search and keyword search.
+
+### Example requests
+
+- [http://o2r.uni-muenster.de/api/v1/search?q=*&from&to&coords&keyword](http://o2r.uni-muenster.de/api/v1/search?q=*&from&to&coords&keyword)
+- [http://o2r.uni-muenster.de/api/v1/search?q=*&from=Wed+Apr+02+2014+00%3A00%3A00+GMT%2B0200+%28CEST%29&to=Fri+Jun+09+2017+00%3A00%3A00+GMT%2B0200+%28CEST%29%0D%0A&coords=%5B%5B%5B-74.33349609375%2C55.44771083630114%5D%2C%5B-74.33349609375%2C83.91787587317228%5D%2C%5B7.22900390625%2C83.91787587317228%5D%2C%5B7.22900390625%2C55.44771083630114%5D%2C%5B-74.33349609375%2C55.44771083630114%5D%5D%5D&keyword=geosciences](http://o2r.uni-muenster.de/api/v1/search?q=*&from=Wed+Apr+02+2014+00%3A00%3A00+GMT%2B0200+%28CEST%29&to=Fri+Jun+09+2017+00%3A00%3A00+GMT%2B0200+%28CEST%29%0D%0A&coords=%5B%5B%5B-74.33349609375%2C55.44771083630114%5D%2C%5B-74.33349609375%2C83.91787587317228%5D%2C%5B7.22900390625%2C83.91787587317228%5D%2C%5B7.22900390625%2C55.44771083630114%5D%2C%5B-74.33349609375%2C55.44771083630114%5D%5D%5D&keyword=geosciences)
```

Comment:
The the example with coordinates, this would be much easier to read if the you leave away some decimal places, i.e. use `-74.33` instead of `-74.33349609375`.

Also, please add an example without URL-encoding, i.e. using `[` etc. for better readability.